mat64/dense.go

```diff
@@ -5,7 +5,10 @@
 package mat64
 
 import (
+	"bytes"
+
 	"github.com/gonum/blas/blas64"
+	"github.com/gonuts/binary"
```

Comment:
@kortschak : I started with a minimal writeBin/writeLen/readBin/readLen (as the commit logs show) then went to gonuts/binary for support of Triangular, SVD, ...